### DISCUSSION TITLE
The basics for installing ebusd on HomeAssistant

### DISCUSSION THREAD

**Discussion by silviopen** [wrongisthenewright/ebusd-configuration-ariston-bridgenet#12] [Q&A]
HI,
I can't find a step by step instructions on how to install and configure ebusd on Home Assistant to read the values ​​of my Elco heat pump. I have a raspberry 4 and the eBUS Adapter Shield v5.
it is right to use the LukasGrebe addon https://github.com/LukasGrebe/ha-addons ?
To communicate directly, do I need to set up communication via USB Device?
In the configuration page I only have this option /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0 and nothing that refers to the ttyAMA0 port as I see indicated on the manufacturer's website https://adapter.ebusd.eu/v5/index.en.html#rpi

**Comment by wrongisthenewright** [comment]
Hi,
I have a different adapter so my answer may non be correct but from what I have seen in the adapter website you should be able to connect via wifi. 
Connect the adapter to the bus (see adapter manual/instructions) 
Connect a USB power source (eg. Mobile phone charger) to the adapter
Connect to the adapter own wifi with your phone and set up the WiFi to connect to your house network
Get the adapter IP address (from your router/access point
Configure the add-on to point at that IP.
Load into the Ebusd addon my ariston.csv and mqtt-hassio.cfg

Enjoy


Having said that please ask the same questions to the adapter and add-on developers.
Came back here after you have the system settled and functioning:)

**Comment by wrongisthenewright** [comment]
Redirected to the adapter and software developers
